apiReference.json

```diff
@@ -0,0 +1,160 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "version": "1.0.0",
+    "title": "Witter",
+    "description": "Wits and user management API",
+    "termsOfService": "http://api_url/terms/",
+    "contact": {
+      "name": "Wolox Team",
+      "email": "daniel.uruena@wolox.co",
+      "url": "https://www.wolox.com.ar/"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "servers": [
+    {
+      "url": "http://localhost:8081/",
+      "description": "Local server"
+    }
+  ],
+  "tags": [
+    {
+      "name": "Users"
+    }
+  ],
+  "paths": {
+    "/users": {
+      "post": {
+        "tags": [
+          "Users"
+        ],
+        "summary": "Sign up user",
+        "description": "Sign up user",
+        "operationId": "signUp",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/User"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "User registered",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/UserResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Missing parameters",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                },
+                "example": {
+                  "message": "the field 'email' is required",
+                  "internal_code": "input_data_error"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "id": {
+        "type": "integer",
+        "description": "Users identifier",
+        "example": 1
+      },
+      "firstName": {
```

Comment:
I think you can define which fields are required in the body

Reply:
fields marked as required, although this mark is not visible.